Mathlib/MeasureTheory/Integral/Bochner.lean

```diff
@@ -872,6 +872,16 @@ theorem ennnorm_integral_le_lintegral_ennnorm (f : α → G) :
 theorem integral_eq_zero_of_ae {f : α → G} (hf : f =ᵐ[μ] 0) : ∫ a, f a ∂μ = 0 := by
   simp [integral_congr_ae hf, integral_zero]
 
+@[simp]
+lemma integral_of_isEmpty {α β : Type*} [MeasurableSpace α] [NormedAddCommGroup β]
+    [NormedSpace ℝ β] [IsEmpty α] (f : α → β) (μ : Measure α) : ∫ x, f x ∂μ = 0 :=
+  integral_eq_zero_of_ae <| eventually_of_forall (IsEmpty.forall_iff.mpr True.intro)
```

Comment:
Thank you.
I'm not sure how useful this would be as a `simp` lemma, or how much that would impact performance. For now I will remove the `simp` attribute.